F/FillArrays/Compat.toml

```diff
@@ -9,3 +9,6 @@ julia = "0.6-1.3"
 
 ["0.3-0.5"]
 julia = "0.7-1.3"
+
+["0.6"]
+julia = "0.7.0-*"
```

Comment:
This is a bold statement.

Reply:
I'm not sure I understand.

Reply:
This claims this version supports Julia 2, Julia 3 etc.